### ISSUE TITLE
Preauricular lymphadenopathy

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10920]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
PMID: 32313895 Atypical manifestation occurs in 5% of the cases and may include Parinaud’s  oculoglandular syndrome (which comprises granulomatous nonsuppurative conjunctivitis with adjacent ipsilateral **preauricular lymphadenopathy**), neuroretinitis, encephalitis, osteomyelitis, endcarditis, hepatosplenic lesions, and fever of unknown origin.

PMID: [38941282] Parinaud oculoglandular syndrome (POGS) is a unilateral granulomatous palpebral conjunctivitis associated with **preauricular**, submandibular, and occasionally cervical **lymphadenopathies**. POGS must be distinguished from Parinaud syndrome, the multicausal triad of impaired upward gaze, convergence retraction nystagmus, and pupillary hyporeflexia. Both syndromes were first reported by Henri Parinaud, a French ophthalmologist (3 cases of POGS in 1889). Several infectious diseases can cause POGS, usually with a conjunctival entry. The characteristic painful or tender lymphadenopathy initially appears in the primary regional lymph node regions of the eyelids. The most common background pathology is cat scratch disease (CSD) caused by Bartonella henselae. 

PMID: 39286673 Henri Parinaud, an ophthalmologist from France first described the oculoglandular syndrome in 1889, which has come to be known as the ocular bartonellosis, describing three patients with fever, regional **lymphadenopathy (preauricular** and submandibular lymph nodes), and a follicular conjunctivitis.

**Parent term (use hpo.jax.org/app)**
Lymphadenopathy HP:0002716

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Cat-scratch disease ORPHA:50839

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6001188
- label: Preauricular lymphadenopathy
- parents: [Lymphadenopathy (HP:0002716)](https://hpo.jax.org/app/browse/term/HP:0002716)
- pmid: PMID:32313895
- def: Enlargement of one or more lymph nodes located in front of the ear.
- comment: 
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10920
- synonyms:
